PROJECTS.md

```diff
@@ -50,3 +50,4 @@ Please see [project lifecycle process](./PROJECT_LIFECYCLE.md) for the project l
 | [Eiffel Sepia](https://github.com/eiffel-community/eiffel-sepia) | | [eiffel-sepia-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-sepia-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
 | [Eiffel Store](https://github.com/eiffel-community/eiffel-store) | A persistence solution for Eiffel events which accepts live view of Eiffel events. | [eiffel-store-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-store-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
 | [Eiffel Vici](https://github.com/eiffel-community/eiffel-vici) | Eiffel Vici dynamically visualizes Eiffel events retrieved from an Eiffel Event Persistence Service. | [eiffel-vici-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-vici-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
+| [Eiffel Goer](https://github.com/eiffel-community/eiffel-goer) | Eiffel Goer is an open source implementation of the event repository API. | [eiffel-goer-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-goer-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
```

Comment:
Would explaining the name help? Could update it to something like
```suggestion
| [Eiffel Goer](https://github.com/eiffel-community/eiffel-goer) | Eiffel Goer is an implementation of the event repository API in GO. | [eiffel-goer-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-goer-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
```

If you get my idea?

Reply:
Done